packages/ds/src/admin/form/InputPhone.tsx

```diff
@@ -0,0 +1,250 @@
+import { classNames } from "@/utils";
+import React from "react";
+
+import { ErrorIcon, ErrorMessage, getFormBaseClass } from "./InputBase";
+import { HiOutlineChevronDown } from "react-icons/hi";
```

Comment:
Je chipotte, mais pour un poil plus de clarté on peut mettre en premier les imports de lib, puis les imports locaux.

```suggestion
import React from "react";
import { HiOutlineChevronDown } from "react-icons/hi";

import { classNames } from "@/utils";
import { ErrorIcon, ErrorMessage, getFormBaseClass } from "./InputBase";
```